ruby_exercises_18_01_26/ruby_exercises_q2.rb

```diff
@@ -0,0 +1,20 @@
+class Book
+
+  def initialize title, chapters
+    @title, @chapters = title, chapters
+  end
+
+  def add_chapter new_chapter
+    @chapters.push new_chapter
+  end
+
+
+  def chapters
+    counter = 1
+    puts "Your book, #{@title}, has #{@chapters.length} chapters:"
```

Comment:
Not sure this works as you expect as counter doesn't increment. Also look at the method each_with_index as you could do `"#{index + 1}. #{c}"`